### DISCUSSION TITLE
How can I setup tab-like pages?

### DISCUSSION THREAD

**Discussion by teijiIshida** [Akascape/CTkDesigner-Support#21] [Q&A] [answered]
I'm trying to add 4 tabs to my app. There is no tabview widget, so it was suggested to use pages. I read the manual, but I still don't understand how to setup tab-like pages.

Here is how my app looks like. The 4 tabs are Main, Esrgan, Denoise, TVAI (using buttons as placeholder). I don't want them all to be the same color. Only the active tab should be blue, while inactive tabs should be grey. (Edit: I figured out that top frame and all widgets can be Fix Above to appear in all pages. I also finally figured out that I can Add Navigation to the buttons to go to different pages). How can I make buttons more tab-like like I described? Or is there a better widget than CTkButton?

@Akascape  Can you write a guide or give a small example? 
![2024-06-06 16_34_27-New Discussion · Akascape_CTkDesigner-Support — Mozilla Firefox](https://github.com/Akascape/CTkDesigner-Support/assets/33733459/3028112c-94fb-4e7e-b071-6f0235e11951)

**Comment by Akascape** [comment, marked as answer]
@teijiIshida 
> How can I make buttons more tab-like like I described? Or is there a better widget than CTkButton?

Currently this is the only way to make tabview. There is ctksegmented but you have to manually write the command code after exporting it to py. 

I will add command for segmented button in next update. 
CtkTabview widget has lots of bugs and not at all flexible like pages. So the method you used is better.

**Comment by teijiIshida** [comment]
Segmented button seems like the way to go. Thank you.

**Reply by Akascape** [reply]
I have implemented it. Wait for the update
